packages/loader/driver-utils/src/blobAggregationStorage.ts

```diff
@@ -312,7 +313,7 @@ export class BlobAggregationStorage extends SnapshotExtractor implements IDocume
             return summary;
         }
         // Only pack at data store level.
-        const startingLevel = level === 1;
+        const startingLevel = level === BlobAggregationStorage.dataStoreLevel;
```

Comment:
What we can do also is instead of calculating this number based on disableIsolatedChannels, we can only enable it for trees that have .component blob. That way it would be more robust RE changes in depth. That said, it would now depend on structure of snapshots. So maybe doing both (asserting that it's one and the same) is best, because it would immediately warn us if depth or structured changed and we can adjust it before cutting release.